.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+<!-- If this is your first pull request: sign the CLA with this GitHub app: https://cla-assistant.io/renovatebot/renovate -->
```

Comment:
Do we need a CLA to be signed for this repository as well?

There's no CI to check if a CLA has been signed on this repo right now.

Reply:
No, not required hhere

Reply:
```suggestion
```